package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "0.14.3",
+  "version": "0.14.4",
```

Comment:
since this is a breaking change, we should run `npm version minor` instead to bump it to 0.15.0
(unless we make this a staged change, in which case 0.14.4 is fine - comment below)